src/app/personside/infotabs/saksoversikt/dokumentvisning/DokumentOgVedlegg.tsx

```diff
@@ -60,12 +61,26 @@ function VisDokumentContainer(props: { fødselsnummer: string; journalpostId: st
     const dokUrl = getSaksdokument(props.fødselsnummer, props.journalpostId, props.dokumentreferanse);
 
     return (
-        <object data={dokUrl} width={'100%'}>
-            <AlertStripeAdvarsel>Du har ikke tilgang til dokument.</AlertStripeAdvarsel>
-        </object>
+        <ObjectHttpFeilHandtering
+            url={dokUrl}
+            width="100%"
+            errorFallback={statusKode => <AlertStripeAdvarsel>{feilmelding(statusKode)}</AlertStripeAdvarsel>}
```

Comment:
Vi har alt ett api for feilhåndtering `onError`, så da tenker jeg vi bare kan gjenbruke det.